bench-tps/src/main.rs

```diff
@@ -162,6 +165,32 @@ fn create_client(
                 ),
             }
         }
+        ExternalClientType::HighTpsClient => {
+            let rpc_client = Arc::new(RpcClient::new_with_commitment(
+                json_rpc_url.to_string(),
+                commitment_config,
+            ));
+            match connection_cache {
+                ConnectionCache::Udp(_) => {
+                    unimplemented!("UDP protocol is not supported.");
+                }
+                ConnectionCache::Quic(cache) => Arc::new(
+                    HighTpsClient::new(
+                        rpc_client,
+                        websocket_url,
+                        HighTpsClientConfig {
+                            fanout_slots: 1,
```

Comment:
Wrote in the other comment my thoughts about that. Regarding configuration in general, I would try to decrease the number of configurable parameters to simplify the ux of the tool. If we know some optimal parameters, I would prefer to hardcode them to decrease the mental to understand how to configure them. Currently, we have: number of threads, tx count, size of CC, sustained/not, timeout, ... I think we should just find one optimal set of parameters because it most of these are never touched. And if they are changed, it might mean that the defaults are not the most performant.